bounties/npm/pathval/1/vulnerability.json

```diff
@@ -0,0 +1,55 @@
+{
+    "PackageVulnerabilityID": 1,
+    "DisclosureDate": "2020-09-13",
+    "AffectedVersionRange": "*",
+    "Summary": "Prototype Pollution",
+    "Contributor": {
+        "Discloser": "Asjidkalam",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "pathval",
+        "URL": "https://github.com/chaijs/pathval/",
+        "Downloads": "11M"
+    },
+    "CWEs": [
+        {
+            "ID": "CWE-400",
+            "Description": ""
+        }
+    ],
+    "CVSS": {
+        "Version": "3.1",
+        "AV": "N",
+        "AC": "H",
+        "PR": "N",
+        "UI": "N",
+        "S": "U",
+        "C": "L",
+        "I": "L",
+        "A": "L",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "5.6"
+    },
+    "CVEs": [
+        ""
+    ],
+    "Repository": {
+        "URL": "https://github.com/chaijs/pathval/",
+        "Codebase": [
+            "JavaScript"
+        ],
+        "Owner": "chaijs",
+        "Name": "pathval",
+        "Forks": "6",
+        "Stars": "37"
+    },
+    "Permalinks": [
+        "https://github.com/chaijs/pathval/blob/master/index.js#L112",
+	      "https://github.com/chaijs/pathval/blob/master/index.js"
```

Comment:
Please remove the link to the entire file

Reply:
Thanks @Mik317,
i have resolved the issue. 😄